docs/csharp/whats-new/breaking-changes.md

```diff
@@ -1,13 +1,190 @@
 ---
 title: Breaking changes in the C# compiler
-description: Find any breaking changes in the C# compiler that you are using.
+description: Find any breaking changes in the C# compiler that you're using.
 ms.topic: troubleshooting
 ms.custom: updateeachrelease
-ms.date: 03/09/2021
+ms.date: 03/04/2022
 ---
 
 # Learn about any breaking changes in the C# compiler
 
+This section explains the breaking changes since C# 10 (.NET 6) general release, including the reason and how to work around the change.
+
+## Disallow converted strings as a default argument
+
+***Introduced in .NET SDK 6.0.300, Visual Studio 2022 version 17.2.***
+
+This change fixes a spec violation in the compiler. Default arguments must be compile time constants. Previous versions allowed the following code:
+
+```csharp
+void M(IEnumerable<char> s = "hello")
+```
+
+The preceding declaration required a conversion from `string` to `IEnumerable<char>`. The compiler allowed this construct, and would emit `null` as the value of the argument.
+
+To work around this change, you can make one of the following changes:
+
+1. Change the parameter type so a conversion isn't required.
+1. Change the value of the default argument to `null` to restore the previous behavior.
+
+## The contextual keyword `var` as an explicit lambda return type
+
+***Introduced in .NET SDK 6.0.200, Visual Studio 2022 version 17.1.***
+
+This change enables [potential future features](https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-06-02.md#lambda-return-type-parsing) by ensuring that the `var` remains the natural type for the return type of a lambda expression.
+
+You can encounter this error if you have a type named `var` and define a lambda expression using an explicit return type of `var` (the type).
+
+```csharp
+using System;
+
+F(var () => default);  // error: 'var' cannot be used as an explicit lambda return type
+F(@var () => default); // ok
+f(() => default);      // OK: return type is inferred from the parameter to F().
```

Comment:
Correct the method name, and a minor suggestion to update the comment to match the previous comments.
```suggestion
F(() => default);      // ok: return type is inferred from the parameter to F()
```